GitPrompt.ps1

```diff
@@ -306,8 +306,4 @@ $PoshGitVcsPrompt = {
     Write-GitStatus $GitStatus
 }
 
-# Install handler for removal/unload of the module
 $Global:VcsPromptStatuses += $PoshGitVcsPrompt
-$ExecutionContext.SessionState.Module.OnRemove = {
-    $Global:VcsPromptStatuses = $Global:VcsPromptStatuses | Where-Object { $_ -ne $PoshGitVcsPrompt }

```

Comment:
BTW I need to update that logic a bit to check if someone might have installed a different prompt function after we installed the posh-git default one.